### PR TITLE
Added users roles add sub command

### PR DESCRIPTION
**Pull request by alijared**
This command is only usable by super users

### CODE REVIEW

File: croud/cmd.py [hunk 1]
```diff
@@ -158,6 +158,18 @@ def org_plan_type_arg(parser: ArgumentParser) -> None:
     )
 
 
+def resource_id_arg(parser: ArgumentParser, required: bool) -> None:
+    parser.add_argument("--resource", type=str, help="Resource ID", required=required)
+
+
+def role_fqn_arg(parser: ArgumentParser) -> None:
+    parser.add_argument("--role", type=str, help="Role FQN", required=True)
```
Comment (mikethebeer) [inline croud/cmd.py]: how is the user informed about the supported FQNs he could use?

Comment (alijared) [reply croud/cmd.py]: There is a GraphQL query for listing all roles, that has an FQN field. I will add the this query to croud as well

Comment (mikethebeer) [reply croud/cmd.py]: @alijared ok - I'm fine with that. However, can we link then somehow in the `help` to use the `list roles` command to get a list of all supported FQNs ?

Comment (alijared) [reply croud/cmd.py]: @mikethebeer I added it to the help text for the command

File: croud/cmd.py [hunk 2]
```diff
@@ -158,6 +158,23 @@ def org_plan_type_arg(parser: ArgumentParser) -> None:
...
 
+def resource_id_arg(parser: ArgumentParser, required: bool) -> None:
+    parser.add_argument("--resource", type=str, help="Resource ID", required=required)
+
+
+def role_fqn_arg(parser: ArgumentParser) -> None:
+    parser.add_argument(
+        "--role",
+        type=str,
+        help="Role FQN. Run `croud roles list` for a list of available roles",
```
Comment (mikethebeer) [inline croud/cmd.py]: 👍

**Pull request merged**